pipeline/load.py

```diff
@@ -21,14 +22,100 @@ def create_database_connection(config: _Environ) -> Connection:
     return conn
 
 
+def insert_into_location_table(connection: Connection, plant_data: pd.DataFrame) -> None:
+    """Inserts data from a pandas data frame into a SQL Server into the location table."""
+
+    plant_list = plant_data.values.tolist()
+
+    for plant in plant_list:
+
+        query = sql.text(
+            "SELECT location_id FROM s_delta.location WHERE region = (:region)")
+        args = ({"region": plant[9]})
+        id = connection.execute(query, args).fetchone()
+
+        if id is None:
+            region = plant[9]
+            continent = plant[10]
+            country = plant[11]
+
+            query = sql.text(
+                """INSERT INTO s_delta.location (region,country,continent)
+                VALUES (:region,:continent,:country)""")
+            connection.execute(
+                query, {"region": region, "continent": continent, "country": country})
+
+
+def insert_into_botanist_table(connection: Connection, plant_data: pd.DataFrame) -> None:
+    """Inserts data from a pandas data frame into a SQL Server into the location table"""
+
+    plant_list = plant_data.values.tolist()
+
+    for plant in plant_list:
+
+        connection.execute(sql.text("USE plants;"))
+
+        query = sql.text(
+            "SELECT botanist_id FROM s_delta.botanist WHERE telephone_number = (:telephone)")
+        args = ({"telephone": plant[8]})
+        id = connection.execute(query, args).fetchone()
+
+        if id is None:
+
+            name = plant[6]
+            email = plant[7]
+            telephone = plant[8]
+
+            query = sql.text(
+                """INSERT INTO s_delta.botanist (name,email,telephone_number)
+                VALUES (:name,:email,:telephone)""")
+            connection.execute(
+                query, {"name": name, "email": email, "telephone": telephone})
+
+
+def insert_into_plant_table(conn: Connection, plant_data: list[dict]) -> None:
+    """Seed the plant table with the plant data list and relevant botanist and location ids."""
+
+    plant_data = plant_data.to_dict('records')
+
+    for plant in plant_data:
+
+        query = sql.text(
+            "SELECT plant_id FROM s_delta.plant WHERE name = (:name)")
+        args = ({"name": plant["Name"]})
+        id = conn.execute(query, args).fetchone()
+
+        if id is None:
+
+            query = sql.text(
+                "SELECT botanist_id FROM s_delta.botanist WHERE telephone_number = (:num)")
```

Comment:
Like Shivani said earlier, maybe change this to email just so everything's standardised!